packages/client-ui/src/components/NetworkAnalysisProject.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+
+import { ProjectType } from '../context/projectContext'
+import useStyles from './NetworkAnalysisProject.styles'
+
+import Button from '@material-ui/core/Button'
+import Paper from '@material-ui/core/Paper'
+import Typography from '@material-ui/core/Typography'
+
+import BackupIcon from '@material-ui/icons/Backup'
+import VisibilityIcon from '@material-ui/icons/Visibility'
+
+import NAGraphContainer from './network-analysis/NAGraphContainer'
+import NAParameters from './network-analysis/NAParameters'
+import DialogModal from './common/DialogModal'
+import NADataUpload from './network-analysis/NADataUploadForm'
+
+type NAProjectProps = {
+  project: ProjectType
+}
+
+const NetworkAnalysisProject = ({ project }: NAProjectProps): JSX.Element => {
```

Comment:
I appreciate it may be temporary here but it still keeps me worried that this nudges the app architecture towards a wrong direction that will be more difficult to untangle later. The UI code should not have reference to particular workflows. All it should know is the structure of the workflow object and be able to render the UI based on that, be that a network analysis workflow or any other workflow.

Reply:
How do you suggest that we name network-analysis-specific components?